wof/examples/flask/odm2/timeseries/odm2_timeseries_dao.py

```diff
@@ -1,3 +1,7 @@
+# -*- coding: utf-8 -*-
+"""
+    odm2_timeseries_dao.py
+"""
```

Comment:
> (I'll disable docstrings tests btw to avoid that. Humans are better than linters to decided when to add them)

Thanks! Hard to really docstring everything.